backend/common/src/node_types.rs

```diff
@@ -38,7 +38,7 @@ pub struct NodeDetails {
     pub validator: Option<Box<str>>,
     pub network_id: NetworkId,
     pub startup_time: Option<Box<str>>,
-    pub target_os: Option<Box<str>>,
+    pub target_os: Option<Box<str>>, 
```

Comment:
Nit: remove unnecessary space?

```suggestion
    pub target_os: Option<Box<str>>,
```